backend/src/baserow/contrib/automation/data_providers/data_provider_types.py

```diff
@@ -112,6 +121,12 @@ def get_data_chunk(
             )
             raise InvalidFormulaContext(message) from exc
 
+        service = parent_node.service.specific
+
+        parent_node_results = service.get_type().prepare_dispatch_result(
```

Comment:
This is not used anymore but as it's implemented and could be useful later I thought it would be fine to keep it.